src/backend/overviewSite/index.html

```diff
@@ -100,15 +100,33 @@
         footer p {
             margin: 0px;
         }
+        .event-holder {
+            cursor: pointer;
 
-        img {
-            height: 160px;
-            width: 310px;
-            max-width: 100%;
         }
-
-        .margin {
```

Comment:
@naveentherocker Revet this, this class is used in the headers.

Reply:
Done @geekyd